src/constants.ts

```diff
@@ -60,4 +60,6 @@ export const AzureClouds = {
 
 export const CommentIdentifiersRegex: RegExp = /^\s*(#|\/{2})/;
 
-export const VariableDefinitionRegex: RegExp = /^\s*@([^\s=]+)\s*=\s*(.+)\s*$/;
\ No newline at end of file
+export const VariableDefinitionRegex: RegExp = /^\s*@([^\s=]+)\s*=\s*(.+)\s*$/;
+
+export const RequestVariableDefinitionRegex: RegExp = /^\s*#{1,}\s+@name\s+(\S+)\s*$/m;
```

Comment:
I think we should also support line starts with `//`, since double slash is also support as line comment indicator.